src/core/linker.cpp

```diff
@@ -658,12 +658,12 @@ void Linker::Resolve(const std::string& name, int Symtype, Module* m, Loader::Sy
 		}
 		else
 		{
-            __debugbreak();//den tha prepei na ftasoume edo
```

Comment:
Best to replace these with `std::abort` so that they trigger a breakpoint in the debugger instead of silent failure